### DISCUSSION TITLE
bitte ergänzen

### DISCUSSION THREAD

**Discussion by Sascha9b** [StevenBlack/hosts#2051] [General]
dealer.spotify.com
(\.|^)https://glersakr\.com/$
(\.|^)https://www\.greatdexchange\.com/jump/next\.php?r=4419791$
(\.|^)https://dating\.idreamed\.com/lp/idreamed/?noexitctapop&noentrypop&noexitblock&logo&uclick=vcpmuqmy&uclickhash=vcpmuqmy-vcpmuqmy-8rfe-0-2tc8-16zw-16j6-6389a6$
a1.tuyaeu.com
(\.|^)https://nmanateex\.top/redirect?tid=914917&optid=924051&uridca=4399886050466843580$
(\.|^)https://poundaroseflight\.com/u282mgam?key=75bc67f6988343a46d48291c004ffda0$
(\.|^)https://voeunblock\.com/oq6287yr5p8a#$

**Comment by StevenBlack** [comment]
Danke @Sascha9b!

Hosts-Dateien funktionieren nur mit Domains, nicht mit URLs.

Auch werden wir `dealer.spotify.com` sicherlich nicht blockieren!

**Reply by Sascha9b** [reply]
Diese habe ich selbst alle geblockt wegen Werbung
